test/sharness/t0021-config.sh

```diff
@@ -71,6 +71,46 @@ test_config_cmd() {
     grep "\"beep2\": false," actual &&
     grep "\"beep3\": false," actual
   '
+
+  test_expect_success "'ipfs config Identity' fails" '
+       test_expect_code 1 ipfs config Identity 2> ident_out
+  '
+
+  test_expect_success "output looks good" '
+       echo "Error: cannot show or change private key through API" > ident_exp &&
+       test_cmp ident_exp ident_out
+  '
+
+  # SECURITY
+  # Those tests are here to prevent exposing the PrivKey on the network
+  test_expect_success "'ipfs config Identity.PrivKey' fails" '
+       test_expect_code 1 ipfs config Identity.PrivKey 2> ident_out
+  '
+
+  test_expect_success "output looks good" '
+       test_cmp ident_exp ident_out
+  '
+
+  test_expect_success "'ipfs config show' doesn't include privkey" '
+       ipfs config show > show_config &&
+       test_expect_code 1 grep PrivKey show_config
+  '
+
+  test_expect_success "'ipfs config replace' injects privkey back" '
```

Comment:
Oh, I added test case for lower case key in the config file, I will add test case for `ipfs config replace` with lower case.